test/repl/env.sh

```
@@ -172,13 +101,9 @@ export TEST_SPIFFE=1
 
 
 # overide these settings in PR
-# $WORKSPACE/test/env.sh
+# $WORKSPACE/tests/env.sh
 if [ -f $WORKSPACE/test/env.sh ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)